src/screens/AddEditObservation/ObservationBase.js

```diff
@@ -89,9 +91,23 @@ const AddEditObservation = props => {
     props.onCurrentDateTime();
   };
 
+  const onCancelExit = () => {
```

Comment:
I would prefer `onCancel` instead of `onCancelExit`. Because we already have `onExit`.

Reply:
FIxed